atomic_reactor/schemas/container.json

```diff
@@ -78,6 +78,10 @@
         },
         "signing_intent": {
           "type": "string"
+        },
+        "inherit_from_parent": {
```

Comment:
Would just "inherit" be a better name? It can only inherit from the parent, after all.

Reply:
sure that should do it